Cesium3DTilesSelection/include/Cesium3DTilesSelection/RasterOverlayCollection.h

```diff
@@ -1,8 +1,10 @@
 #pragma once
 
-#include "Library.h"
-#include "RasterOverlay.h"
-#include "RasterOverlayTileProvider.h"
+#include <Cesium3DTilesSelection/Library.h>
```

Comment:
These were using quotes before because they were relative paths to files in the same directory as this header file. The idea being that Cesium3DTilesSelection should include other parts of itself from the same directory, rather than resolving `Cesium3DTilesSelection` anew. I don't have strong feelings on this - it's pretty unlikely to make a difference to anyone - but we should try to be consistent about it.